config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
 
+  # route is /healthcheck, see config/initializers/healthcheck.rb to configure
```

Comment:
Adding based on feedback from [another PR](https://github.com/WPMedia/klaxon/pull/3#discussion_r1056619052) because Ruby is a mysterious mystery ❓❗ 🤔